bin/patch

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env perl
+# patch - Apply a patch to Webmin core or its modules from GitHub or a local file
+
+use strict;
+use warnings;
+
+use 5.010;
+
+use Getopt::Long qw(:config permute pass_through);
+use Pod::Usage;
+use Term::ANSIColor qw(:constants);
```

Comment:
Is this dependency needed?

Reply:
Yeah, we use it everywhere for colors ..

Reply:
But I see no reference to `constants` in this PR? Also, I strongly prefer that we do not add coloring to the output of commands, as it messes up when they are fed into grep or copy/pasted in some terminals.

Reply:
Thanks!

I missed this comment. Alright, I removed `Term::ANSIColor` dependency.